fennel/client_tests/branches/test_create.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+
+import pandas as pd
+import pytest
+
+from fennel._vendor import requests
+from fennel.datasets import dataset, field
+from fennel.featuresets import featureset, feature
+from fennel.sources import source, Webhook
+from fennel.test_lib import mock
+
+wh = Webhook(name="fennel_webhook")
+__owner__ = "fennel <<EMAIL>>"
```

Comment:
This should not be allowed? it should always be an email regex

Reply:
got it.